code/game/machinery/lightswitch.dm

```diff
@@ -2,9 +2,13 @@
 /obj/machinery/light_switch
 	name = "light switch"
 	icon = 'icons/obj/power.dmi'
-	icon_state = "light1"
+	icon_state = "light-p"
 	desc = "Make dark."
 	power_channel = LIGHT
+
+	light_power = 0
+	light_range = 7
```

Comment:
?

Reply:
This allows the lightswitch object to be visible even in darkness. The light_range var makes its "light" and therefore its sprite visible at a distance of 7, but since light_power is 0 it doesn't actually emit any light. Without them, the lightswitch would glow in the dark but only if you were directly adjacent to it.